h/cli/commands/devdata.py

```diff
@@ -0,0 +1,164 @@
+"""
+A CLI command for inserting standard dev data into the DB.
+
+Usage:
+
+    bin/hypothesis --dev devdata
+"""
+import click
+import json
+import os.path
+import pathlib
+import shutil
+import subprocess
+import tempfile
+
+import h
+from h import models
+
+
+class DevDataFactory:
+    """Factory class for inserting standard dev data into the DB."""
+
+    def __init__(self, request, devdata):
+        self.db = request.db
+        self.tm = request.tm
+        self.devdata = devdata
+
+        self.user_password_service = request.find_service(name="user_password")
+        self.group_service = request.find_service(name="group")
+        self.group_create_service = request.find_service(name="group_create")
+
+    def create_all(self):
+        """
```

Comment:
I think it would be useful here to have some logging indicating the changes that were made by a given run of `make devdata`, ie. which entities were created or modified.